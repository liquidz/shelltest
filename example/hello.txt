bash$ echo hello shelltest
hello shelltest
bash$ echo $?
0
