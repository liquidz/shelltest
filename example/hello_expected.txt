$ shelltest example/hello.txt
..

2 tests, 0 failures
